CMakeLists.txt

```diff
@@ -74,11 +74,18 @@ subdirlist(TEST_CATEGORIES ${TESTDIR})
 foreach(SUPER_SLAB_SIZE 1;16)
   foreach(TEST_CATEGORY ${TEST_CATEGORIES})
     subdirlist(TESTS ${TESTDIR}/${TEST_CATEGORY})
+    if(MSVC)
```

Comment:
Is this required?

Reply:
If you include malloc.cc in the test, does it do the right thing?

Reply:
From CI it looks like it doesn't.  Leave the exception, and I can fix this later.

Reply:
The CI says no

Reply:
Should we be providing symbols like `_msize` and `_aligned_malloc`? If so, do we support all of these: https://docs.microsoft.com/en-us/cpp/c-runtime-library/data-alignment?view=vs-2017?

Reply:
ok, I'll leave this for later